src/runtime_src/tools/xclbinutil/XclBinClass.cxx

```diff
@@ -16,96 +16,103 @@
 
 // ------ I N C L U D E   F I L E S -------------------------------------------
 #include "XclBinClass.h"
+
+#include "ElfUtilities.h"
+#include "FormattedOutput.h"
+#include "KernelUtilities.h"
 #include "Section.h"
+#include "version.h"                            // Generated include files
+#include "XclBinUtilities.h"
 
-#include <stdexcept>
-#include <boost/property_tree/json_parser.hpp>
+#include <algorithm>
 #include <boost/algorithm/string.hpp>
-
-#include <boost/uuid/uuid.hpp>          // for uuid
-#include <boost/uuid/uuid_io.hpp>       // for to_string
 #include <boost/filesystem.hpp>
+#include <boost/format.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/uuid/uuid.hpp>                  // for uuid
+#include <boost/uuid/uuid_io.hpp>               // for to_string
 #include <random>
 #include <sstream>
-#include <boost/format.hpp>
+#include <stdexcept>
+#include <stdlib.h>
 
-#include "XclBinUtilities.h"
-#include "KernelUtilities.h"
-namespace XUtil = XclBinUtilities;
+// Constant data
+static const std::string MIRROR_DATA_START("XCLBIN_MIRROR_DATA_START");
+static const std::string MIRROR_DATA_END("XCLBIN_MIRROR_DATA_END");
```

Comment:
lower case variable names is better, in general ALL_UPPER_CASE should be reserved for macros, which we shouldn't really use ;_)

Reply:
Renamed the string to be camelCase.